platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/main/java/com/swirlds/platform/test/consensus/framework/validation/InputEventsValidation.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.test.consensus.framework.validation;
+
+import com.swirlds.platform.internal.EventImpl;
+import com.swirlds.platform.test.consensus.framework.ConsensusOutput;
+import java.util.List;
+import java.util.Objects;
+import org.junit.jupiter.api.Assertions;
+
+public class InputEventsValidation {
+    /**
+     * Validate that the events are added in a different order
+     */
+    public static void validateEventsAreInDifferentOrder(final ConsensusOutput output1, final ConsensusOutput output2) {
```

Comment:
Nullity markings on arguments in whole file.

Reply:
added